presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,7 +14,90 @@ make it executable with ``chmod +x``, then run it:
 
     ./presto --server localhost:8080 --catalog hive --schema default
 
-Run the CLI with the ``--help`` option to see the available options.
+The available options for the CLI are:
```

Comment:
We checked with @martint on that on Slack and he said: "I like being able to go to a web page to see what the options are (similar to the web-based "man" pages for linux commands), so that would make sense to me."
I understand that any docs can go out of date in general; is there any reason why the CLI might be more prone to that?